script.js

```diff
@@ -0,0 +1,51 @@
+$(document).ready(function() {
+    $('#action').click(function() {
+	//DO NOT CHANGE CODE BELOW
+        var num1 = parseInt($('#num1').val());
+        var num2 = parseInt($('#num2').val());
+        clear();
+        addNumbers(num1, num2);
+        var result = multiplyNumbers(num1, num2);
+        $('#result-mult').text(result);
+	//DO NOT CHANGE CODE ABOVE
+
+    });
+
+	/*
+	 Below this comment, create a function 
+	 named addNumbers, which accepts two parameters.
+	 The function should add the two parameters together
+	 and write the result to the element with the id
+	 result-add
+    */
+    function addNumbers(num1,num2){
+        document.getElementById("num1","num2").innerHTML=addNumbers;
+        $("#result-add").append('<div></div>');
+        var(result-add) = num1 + num2;
```

Comment:
no parentheses needed. so here we would have the variable result-add that holds the sum - we would then need to write that to #result-add. Instead you return the value. So take out line 25 and replace it with soemthing like:
$('#result-add').text(result-add);